samples/solution-counters.js

```diff
@@ -0,0 +1,72 @@
+/*!
+ * Copyright 2019 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+const { Firestore, FieldValue } = require('@google-cloud/firestore');
+
+async function main() {
+  // [START create_counter]
+  async function createCounter(docRef, numShards) {
+    const colRef = docRef.collection("shards");
+    for (let count = 0; count < numShards; count++) {
+      const newDoc = colRef.doc(count.toString());
+      await newDoc.set({ count: 0 });
+    }
+  }
+  // [END create_counter]
```

Comment:
Better write `fs_` prefix in tag names. Well, that's how Ben Whitehead [recommended](https://github.com/GoogleCloudPlatform/golang-samples/pull/959#discussion_r318639468), I know, that this prefix is not "worldwide"

Reply:
Thanks for your feedback!! in above case all samples are in one repository while here it's firestore specific, so I don't think we need to add prefix.